content/en/docs/instrumentation/erlang/getting-started/phoenix.md

```diff
@@ -0,0 +1,271 @@
+---
+title: Phoenix
+weight: 2
+---
+
+This guide will show you how to get started with OpenTelemetry in the Phoenix
+Web Framework.
+
+You will learn how you can instrument a simple application automatically, in
+such a way that traces are emitted to the console.
+
+## Prerequisites
+
+Ensure that you have erlang, elixir, postgres (or the database of your choice),
+and phoenix installed locally. The phoenix
+[installation guide](https://hexdocs.pm/phoenix/installation.html) will help you
+get set up with everything you need.
+
+## Example Application
+
+The following example uses a basic [Phoenix](https://www.phoenixframework.org/)
+web application. For more elaborate examples, see
+[examples](/docs/instrumentation/erlang/examples/).
+
+To begin, use the `phx` generator to create a new project
+
+```shell
+mix phx.new your_app_name
```

Comment:
I think instead of `your_app_name` this should just use the example app name that'll go in the -contrib repo examples.

Reply:
Question about this: there's a [basic_phoenix_ecto](https://github.com/open-telemetry/opentelemetry-erlang-contrib/tree/main/examples/basic_phoenix_ecto), I assume this would be a separate `dice` (`dice_game`?) example in addition to that one?

Reply:
Yup, a separate dice game example.